src/av2/datasets/tbv/README.md

```diff
@@ -15,12 +15,29 @@ The total dataset amounts to 15.57 hours of driving data, amounting to 922 GB of
 
 ## Downloading TbV
 
-Install `s5cmd` using [the installation instructions here](../../../../../DOWNLOAD.md), and then run download 21 tar.gz archives from Amazon S3 as follows:
+TbV is available for download in two forms -- either zipped up as 21 tar.gz files -- or in extracted, unzipped form (without tar archives). Downloading either will produce the same result (the underlying log data is identical).
+
+Using the `tar.gz` files is recommended (depending upon your connection, this is likely faster, as there are almost 8 million images files in the extracted format). We recommend using `s5cmd` to pull down all 21 `.tar.gz` files with a single command. You can see the links to the `tar.gz` files on [the Argoverse 2 downloads page](https://www.argoverse.org/av2.html#download-link).
+
```

Comment:
Just curious --- do we have an estimate on the time difference between these two methods?

Reply:
@benjaminrwilson I haven't timed this yet, but I'll check and get back to you.